src/commands/install.yml

```diff
@@ -31,27 +31,32 @@ steps:
   - run:
       name: "Install Go"
       command: |
-        if which go;then
-          if go version | grep "<< parameters.version >>";then
+        if command -v go &>/dev/null;then
+          if go version | grep -q -F "go<< parameters.version >> ";then
             echo "Binary already exists, skipping download."
             exit 0
           else
             echo "Go is already install but is the wrong version."
             echo "Installing the request version instead."
           fi
 
-          echo "Installing the requested version of Go."
-        fi
+          GOHOSTARCH="$(go env GOHOSTARCH)"
+          GOHOSTOS="$(go env GOHOSTOS)"
 
-        if [[ $OSD_FAMILY == "linux" ]];then
-          curl -sSL "https://dl.google.com/go/go<< parameters.version >>.linux-amd64.tar.gz" | sudo tar -xz -C /usr/local/
-        elif [[ $OSD_FAMILY == "darwin" ]];then
-          curl -sSL "https://dl.google.com/go/go<< parameters.version >>.darwin-amd64.tar.gz" | sudo tar -xz -C /usr/local/
+          $SUDO rm -rf /usr/local/go &>/dev/null
+          $SUDO install --owner=${USER} -d /usr/local/go
```

Comment:
Little confused where this `install` command is coming from, not sure it will work as expected.

Reply:
I'm also a little bit confused as to why this is being done in the first place.

Reply:
Thanks for the opportunity to clarify.

Without this we'd run into overwriting an existing installation and the risk of mixing old files (that remain because they've been deleted or moved in a more recent version) with new ones. Kinda the point of the entire exercise.

As to `install`, it's part of the *GNU coreutils*. If it's no reduced imaged (and even then you refrain) it's supposed to be part of any GNU/Linux installation.

Reply:
Interesting - I had no idea! Disregard this then :)